zkevm-circuits/src/sha256_circuit.rs

```diff
@@ -71,10 +71,19 @@ impl<F: Field> SHA256Circuit<F> {
     fn expected_rows(&self) -> usize {
         self.0
             .iter()
-            .map(|evnt| (evnt.input.len()) + 9 / BLOCK_SIZE_IN_BYTES + 1)
+            .map(|evnt| {
+                let blks = (evnt.input.len() + BLOCK_SIZE_IN_BYTES - 1) / BLOCK_SIZE_IN_BYTES;
+                let paddings = blks * BLOCK_SIZE_IN_BYTES - evnt.input.len();
+                if paddings < 9 {
+                    blks + 1
+                } else {
+                    blks
+                }
```

Comment:
```suggestion
                (evnt.input.len() + LENGTH_BYTES + BLOCK_SIZE_IN_BYTES - 1) / BLOCK_SIZE_IN_BYTES
```
We can simplify the logic a bit and add a constant variable 
```
const LENGTH_BYTES: usize = 8;
```

Reply:
ok. I think the `LENGTH_BYTES` should be 9 instead of 8 (1 zero byte along with `int64`)